target/linux/ath79/dts/ar934x_fortinet_loader.dtsi

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/mtd/partitions/uimage.h>
+
+/ {
+	virtual_flash {
+		compatible = "mtd-concat";
+
+		devices = <&fwconcat0 &fwconcat1>;
```

Comment:
Does &fwconcat2 need to be added here?

Reply:
it's redefined in the DTS

I'm doing it this way just like the DTSI for Senao Engenius boards because it might be used for a device that doesn't have that extra partition

Reply:
watchguard AP300 is an example 

146aaeafb

Reply:
So it is. I missed that, sorry!